code/__DEFINES/dcs/signals/signals_tram.dm

```diff
@@ -3,3 +3,6 @@
 
 /// Sent from /obj/structure/industrial_lift/tram when it begins to travel. (obj/effect/landmark/tram/from_where, obj/effect/landmark/tram/to_where)
 #define COMSIG_TRAM_TRAVEL "tram_travel"
+
+/// Sent from /obj/structure/industrial_lift/tram when it hits someone
```

Comment:
```suggestion
/// Sent from /obj/structure/industrial_lift/tram when it hits someone: ()
```
None but me cares but there is actually syntax to signal comments, it's all the args but source